### DISCUSSION TITLE
Scale out for market node to n+1 in expectation of large influx of storage deals

### DISCUSSION THREAD

**Discussion by stuberman** [filecoin-project/boost#392] [Ideas]
## Improvement Suggestion

Prioritize the scalability of the new market node (v2/Boost/Ignite) to allow more than one market node to be attached to lotus-miner node in order to process more incoming deals. Today we are bottlenecked by a single market node on a given lotus-miner system. We are preparing to accept new deals that are in the PiB order of magnitude. Today a single market node cannot handle more than dozens of TiBs per day. The only solution today to process 100's of TiBs per day is to add more independent Lotus Miner node which is very inefficient.

**Comment by TippyFlitsUK** [comment]
Hey Stu

Thanks for the feature request. I will add labels and will assign it to the right team for analysis.
We might tag you again for more information.

**Comment by jennijuju** [comment]
I would consider transferring this ticket  to boost. As of the next two quarters, lotus is not planning to prioritize market scalability.

**Comment by LexLuthr** [comment]
**FYI(No action needed)**
@stuberman I have converted the discussion to the issue based on the https://boost.filecoin.io/need-help
Boost team would like to start all enhancement and feature requests in discussion. After everyone weighs in, we can take a decision on accepting the request. This process might change after some time.
You can answer the team's questions and add any further inputs here.

**Reply by brendalee** [reply]
@LexLuthr can you link the issue? 

this is also related: https://github.com/storage-provider-association/fc-sp-priority-list_lotus/blob/master/features/multiple-market-nodes.md

**Reply by LexLuthr** [reply]
https://github.com/filecoin-project/boost/issues/464
